quickstart/public/assets/css/common.css

```
@@ -60,7 +60,7 @@ body {
     padding-bottom: 2rem;
 }
 
-hanko-auth, hanko-profile {
+hanko-auth, hanko-profile, hanko-logout {
```

`hanko-logout` can be removed:

```suggestion
hanko-auth, hanko-profile {
```

---

@bjoern-m done.